slatedb/src/clone.rs

```diff
@@ -619,21 +619,18 @@ mod tests {
         test_utils::seed_database(&parent_db, &sample::table(&mut rng, 100, 10), false)
             .await
             .unwrap();
-        parent_db.flush().await.unwrap();
+        parent_db.inner.flush_memtables().await.unwrap();
 
         test_utils::seed_database(&parent_db, &sample::table(&mut rng, 100, 10), false)
             .await
             .unwrap();
-        parent_db.flush().await.unwrap();
+        parent_db.inner.flush_memtables().await.unwrap();
         parent_db.close().await.unwrap();
 
-        fail_parallel::cfg(
-            Arc::clone(&fp_registry),
-            "copy-wal-ssts-io-error",
-            "1*off->return",
-        )
-        .unwrap();
+        fail_parallel::cfg(Arc::clone(&fp_registry), "copy-wal-ssts-io-error", "return").unwrap();
```

Comment:
since `last_seen_wal_id` is only updated on memtable is flushed to L0, the value of `last_seen_wal_id` will become not different a lot from `replay_from_wal_id`. as a result, the replayed ssts in `copy_wal_sst` will become smaller, it makes it harder to reproduce this with `1*off->return`.. i guess this test case might be better to have some further update.

Reply:
Mind updating in this PR? Given how critical this logic is, I'd like to play it safe and make things clean.